docs/en/getting-started/example-datasets/reddit-comments.md

```diff
@@ -0,0 +1,636 @@
+```
+
+A shoutout to Percona for the [motivation behind ingesting this dataset](https://www.percona.com/blog/big-data-set-reddit-comments-analyzing-clickhouse/), which we have downloaded and stored in an S3 bucket.
+
+:::note
+The following commands were executed on ClickHouse Cloud. To run this on your own cluster, replace `default` in the `s3Cluster` function call with the name of your cluster. If you do not have a cluster, then replace the `s3Cluster` function with the `s3` function.
+:::
+
+1. Let's create a table for the Reddit data:
+
+```sql
+CREATE TABLE reddit
+(
+    subreddit LowCardinality(String),
+    subreddit_id LowCardinality(String),
+    subreddit_type Enum('public' = 1, 'restricted' = 2, 'user' = 3, 'archived' = 4, 'gold_restricted' = 5, 'private' = 6),
+    author LowCardinality(String),
+    body String CODEC(ZSTD(6)),
+    created_date Date DEFAULT toDate(created_utc),
+    created_utc DateTime,
+    retrieved_on DateTime,
+    id String,
+    parent_id String,
+    link_id String,
+    score Int32,
+    total_awards_received UInt16,
+    controversiality UInt8,
+    gilded UInt8,
+    collapsed_because_crowd_control UInt8,
+    collapsed_reason Enum('' = 0, 'comment score below threshold' = 1, 'may be sensitive content' = 2, 'potentially toxic' = 3, 'potentially toxic content' = 4),
+    distinguished Enum('' = 0, 'moderator' = 1, 'admin' = 2, 'special' = 3),
+    removal_reason Enum('' = 0, 'legal' = 1),
+    author_created_utc DateTime,
+    author_fullname LowCardinality(String),
+    author_patreon_flair UInt8,
+    author_premium UInt8,
+    can_gild UInt8,
+    can_mod_post UInt8,
+    collapsed UInt8,
+    is_submitter UInt8,
+    _edited String,
+    locked UInt8,
+    quarantined UInt8,
+    no_follow UInt8,
+    send_replies UInt8,
+    stickied UInt8,
+    author_flair_text LowCardinality(String)
+)
+ENGINE = MergeTree
+ORDER BY (subreddit, created_date, author);
+```
+
+:::note
+The names of the files in S3 start with `RC_YYYY-MM` where `YYYY-MM` goes from `2005-12` to `2023-02`. The compression changes a couple of times though, so the file extensions are not consistent. For example:
+
+- the file names are initially `RC_2005-12.bz2` to `RC_2017-11.bz2`
+- then they look like `RC_2017-12.xz` to `RC_2018-09.xz`
+- and finally `RC_2018-10.zst` to `RC_2023-02.zst`
+:::
+
+2. We are going to start with one month of data, but if you want to simply insert every row - skip ahead to step 8 below. The following file has 86M records from December, 2017:
+
+```sql
+INSERT INTO reddit
+    SELECT *
+    FROM s3Cluster(
+        'default',
+        'https://clickhouse-public-datasets.s3.eu-central-1.amazonaws.com/reddit/original/RC_2017-12.xz',
```

Comment:
It does not make sense to use `s3Cluster` with a single file.
The `s3Cluster` is effective only with multiple files.